File: app/src/main/java/com/tunjid/rcswitchcontrol/utils/SpanCountCalculator.java
package com.tunjid.rcswitchcontrol.utils;

import android.util.DisplayMetrics;

import com.tunjid.rcswitchcontrol.App;

public class SpanCountCalculator {

    public static int getSpanCount() {
        DisplayMetrics metrics = App.getInstance().getResources().getDisplayMetrics();
        return metrics.widthPixels > metrics.heightPixels ? 2 : 1;
    }
}

File: app/src/main/java/com/tunjid/rcswitchcontrol/utils/SpanCountCalculator.kt
package com.tunjid.rcswitchcontrol.utils

import com.tunjid.rcswitchcontrol.App

object SpanCountCalculator {

    val spanCount: Int
        get() {
            val metrics = App.instance.resources.displayMetrics
            return if (metrics.widthPixels > metrics.heightPixels) 2 else 1
        }
}
